docs/getting-started-guides/kubeadm.md

```diff
@@ -87,22 +90,21 @@ For each host in turn:
       gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg
              https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
       EOF
-      # setenforce 0
       # yum install -y docker kubelet kubeadm kubectl kubernetes-cni
       # systemctl enable docker && systemctl start docker
       # systemctl enable kubelet && systemctl start kubelet
 
 The kubelet is now restarting every few seconds, as it waits in a crashloop for `kubeadm` to tell it what to do.
 
-Note: `setenforce 0` will no longer be necessary on CentOS once [#33555](https://github.com/kubernetes/kubernetes/pull/33555) is included in a released version of `kubeadm`.
```

Comment:
Ok it does not look fully fixed, we took care of problems with discovery and etcd pods, but if you get to deploying a pod network solution things start to break, problems with /etc/hosts etc. Paul's PR should help with this but I don't think it's in kubelet 1.4.4. I hope to test with master soon and see if it helps with the problems that show up after deploying weave.

I would keep the selinux disable for now.